charts/datadog/values.yaml

```diff
@@ -1121,6 +1127,12 @@ clusterAgent:
         #       cpu: "2"
         #       memory: "1024Mi"
 
+    autoInstrumentation:
+
+      # clusterAgent.admissionController.autoInstrumentation.containerRegistry -- Override the default registry for autoInstrumentation.
+      containerRegistry:
```

Comment:
All the options about apm instrumentation are under `apm.instrumentation` so maybe it's better to include the registry there as well.